modules/manage/pages/iceberg/iceberg-schema-changes-and-migration-guide.adoc

```diff
@@ -0,0 +1,145 @@
+
+. Before upgrading to v25.3, disable Iceberg on all Iceberg topics by setting the `redpanda.iceberg.mode` topic property to `disabled`. This step ensures that no additional Parquet files are written by Iceberg topics.
+. xref:upgrade:rolling-upgrade.adoc#perform-a-rolling-upgrade[Perform a rolling upgrade] to v25.3, restarting the cluster in the process.
+. Query the `GetCoordinatorState` Admin API endpoint repeatedly for these Iceberg topics to migrate to the new schema, until there are no more pending entries in the coordinator for the given topics. This step confirms that all Parquet files written pre-upgrade have been committed to the Iceberg tables.
++
+[,bash]
+----
+# Pass the comma-separated list of Iceberg topics into "topics_filter" 
+curl -s \
+    --header 'Content-Type: application/json' \
+    --data '{"topics_filter": ["<list-of-topics-to-migrate>"]}' \
+    localhost:9644/redpanda.core.admin.internal.datalake.v1.DatalakeService/GetCoordinatorState | jq
+----
++
+.Sample output
+[,bash,.no-copy]
+----
+{
+  "state": {
+    "topicStates": {
+      "topic_to_migrate": {
+        "revision": "9",
+        "partitionStates": {
+          "0": {
+            "pendingEntries": [
+              {
+                "data": {
+                  "startOffset": "12",
+                  "lastOffset": "15",
+                  "dataFiles": [
+                    {
+                      "remotePath": "redpanda-iceberg-catalog/redpanda/topic_to_migrate/data/0-871734c9-e266-41fa-a34d-2afba2828c0d.parquet",
+                      "rowCount": "4",
+                      "fileSizeBytes": "1426",
+                      "tableSchemaId": 0,
+                      "partitionSpecId": 0,
+                      "partitionKey": []
+                    }
+                  ],
+                  "dlqFiles": [],
+                  "kafkaProcessedBytes": "289"
+                },
+                "addedPendingAt": "6"
+              }
+            ],
+            "lastCommitted": "11"
+          }
+        },
+        "lifecycleState": "LIFECYCLE_STATE_LIVE",
+        "totalKafkaProcessedBytes": "79"
+      }
+    }
+  }
+}
+----
++
+To check for remaining pending files:
++
+[,bash]
+----
+curl -s \
+    --header 'Content-Type: application/json' \
+    --data '{}' \
+    localhost:9644/redpanda.core.admin.internal.datalake.v1.DatalakeService/GetCoordinatorState \
+    | jq  '[.state.topicStates[].partitionStates[].pendingEntries | length] | any(. > 0)'
+----
++
+If the query returns `true`, there are pending files and you need to wait longer before proceeding to the next step.
+
+. Migrate Iceberg topics to the new schema translation and ensure they are conformant with the breaking change.
++
+Run SQL queries to rename affected columns for each Iceberg table you want to migrate to the new schema. In addition to renaming the existing columns, Redpanda automatically adds new columns that use the original name, but with the new types:
```

Comment:
Maybe it's also worth explicitly calling out that users must do this for their topics, and pointing again at the table of breaking changes?